Document-Processing/PDF/Conversions/HTML-To-PDF/NET/features.md

```diff
@@ -807,6 +807,101 @@ padding-left: 5px;
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/HTML%20to%20PDF/Blink/Create-custom-style-TOC-when-converting-HTML-to-PDF).
 
+## Skip Cover Page in TOC Numbering
+
+The following code snippet demonstrates how to set the starting page number for the Table of Contents (TOC) so that it skips the cover page. In this example, we have only one cover page, so the starting page number is set to 1. If your document includes multiple cover pages, you can adjust this value accordingly.

```

Comment:
This code snippet shows how to configure the Table of Contents (TOC) to skip the cover page by setting the starting page number. In this example, the cover page is a single page, so the TOC begins at page 1. For documents with multiple cover pages, adjust the starting page number as needed.